task_processing/plugins/kubernetes/kubernetes_pod_executor.py

```diff
@@ -388,7 +471,7 @@ def _process_pod_event(self, event: PodEvent) -> None:
             elif event["type"] == "DELETED":
                 self.__handle_deleted_pod_event(event)
 
-            elif event["type"] == "MODIFIED":
+            elif event["type"] == "MODIFIED" or event["type"] == "ADDED":
```

Comment:
```suggestion
            elif event["type"] in {"MODIFIED", "ADDED"}:
```
would be a little more idiomatic/shorter, but the current code works as well :)